drivers/soundwire/cadence_master.c

```diff
@@ -283,6 +283,29 @@ static int cdns_config_update(struct sdw_cdns *cdns)
 	return ret;
 }
 
+/**
+ * sdw_cdns_config_update() - Update configurations
+ * @cdns: Cadence instance
+ */
+void sdw_cdns_config_update(struct sdw_cdns *cdns)
+{
+	/* commit changes */
+	cdns_writel(cdns, CDNS_MCP_CONFIG_UPDATE, CDNS_MCP_CONFIG_UPDATE_BIT);
```

Comment:
do we need to use cdns_updatel ?

Reply:
It's not necessary @RanderWang 

CONFIG_UPDATE only has BIT(0) defined, bits 1..31 are reserved. So it's useless to have a read-modify-write.

The only times where a read is necessary is *after* a write, when making sure that the bit was indeed cleared by hardware.

We can actually clean-up all other usages of CONFIG_UPDATE, I'll add a patch for this to be consistent. 

Thanks for the feedback.